src/test/java/com/wikia/webdriver/common/remote/notifications/MarkAsRead.java

```diff
@@ -0,0 +1,26 @@
+package com.wikia.webdriver.common.remote.notifications;
+
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.remote.operations.http.PostRemoteOperation;
+import org.json.JSONObject;
+
+class MarkAsRead {
+
+  private final static String MARK_AS_READ_SUFFIX = "mark-all-as-read";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)